components/dash-html-components/scripts/generate-components.js

```diff
@@ -280,8 +319,8 @@ const ${Component} = (props) => {
                 n_clicks_timestamp: Date.now()
             })
             })}
-            {...omit(['n_clicks', 'n_clicks_timestamp', 'loading_state', 'setProps', 'disable_n_clicks'], props)}
-            {...dataAttributes}
+            {...omit(['n_clicks', 'n_clicks_timestamp', 'loading_state', 'setProps', 'disable_n_clicks', 'href'], props)}
```

Comment:
not always `href` though, right?

Reply:
Actually not necessary to omit the prop since it is redefined in extraProps it will take precedence.